### PR TITLE
change json files to support v1beta3

### PR DESCRIPTION
**Pull request by sub-mod**
@vishh kindly review

### CODE REVIEW

**Comment by vishh** [comment]
ok to test

File: deploy/grafana-service.yaml [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
-id: "monitoring-grafana"
-port: 80
-containerPort: 80
-selector: 
-  name: "influxGrafana"
+apiVersion: v1beta3
+kind: Service
+metadata: 
+  labels: 
+    name: influxGrafana
```
Comment (vishh) [inline deploy/grafana-service.yaml]: Why add labels to the service?

Comment (sub-mod) [inline deploy/grafana-service.yaml]: Isn't it a good practice to have the selector as a label in the service too, I saw it in the  discussions on some PR

**Comment by vishh** [comment]
LGTM. Will merge once integration tests pass. Thanks for the PR @sub-mod!!!!

**Comment by sub-mod** [comment]
write error: No space left on device
godep: go exit status 2

**Comment by vishh** [comment]
Fixed that issue and restarted the integration tests. Sorry about the delay
@sub-mod!

**Comment by vishh** [comment]
@sub-mod: You have to rebase to HEAD

**Comment by sub-mod** [comment]
@vishh done

**Comment by vishh** [comment]
LGTM.

**Pull request merged**